samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/cloudbuild.yaml

```diff
@@ -0,0 +1,138 @@
+
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      [
+        "build",
+        "-t",
+        "${_GCR_PATH}/kaggle_visualize_html:$COMMIT_SHA",
+        "-t",
+        "${_GCR_PATH}/kaggle_visualize_html:latest",
+        "${_CODE_PATH}/visualize_html",
+        "-f",
+        "${_CODE_PATH}/visualize_html/Dockerfile",
+      ]
+    id: "BuildVisualizeHTMLImage"
+
+
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      [
+        "build",
+        "-t",
+        "${_GCR_PATH}/kaggle_train:$COMMIT_SHA",
+        "-t",
+        "${_GCR_PATH}/kaggle_train:latest",
+        "${_CODE_PATH}/train_model",
+        "-f",
+        "${_CODE_PATH}/train_model/Dockerfile",
+      ]
+    id: "BuildTrainImage"
+
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      [
+        "build",
+        "-t",
+        "${_GCR_PATH}/kaggle_submit:$COMMIT_SHA",
+        "-t",
+        "${_GCR_PATH}/kaggle_submit:latest",
+        "${_CODE_PATH}/submit_result",
+        "-f",
+        "${_CODE_PATH}/submit_result/Dockerfile",
+      ]
+    id: "BuildSubmitImage"
+
+  - name: "python:3.7-slim"
+    entrypoint: "/bin/sh"
+    args: [
+        "-c",
+        "cd ${_CODE_PATH};
+        pip3 install cffi==1.12.3 --upgrade;
+        pip3 install kfp==0.1.38;
+        python pipeline.py;
+        cp pipeline.py.zip /workspace/pipeline.zip",
+      ]
+    id: "KagglePackagePipeline"
+
+  - name: "gcr.io/cloud-builders/gsutil"
+    args:
+      [
+        "cp",
+        "/workspace/pipeline.zip",
+        "gs://${_GS_BUCKET}/$COMMIT_SHA/pipeline.zip"
+      ]
+    id: "KaggleUploadPipeline"
+    waitFor: ["KagglePackagePipeline"]
+
+
+  - name: "gcr.io/cloud-builders/kubectl"
+    entrypoint: "/bin/sh"
+    args: [
+        "-c",
+        "cd ${_CODE_PATH};
+        apt-get update;
+        apt-get install -y python3-pip;
+        apt-get install -y libssl-dev libffi-dev;
+        /builder/kubectl.bash;
+        pip3 install kfp==0.1.38;
+        pip3 install kubernetes;
+        python3 create_pipeline_version_and_run.py 
+        --package_url https://storage.googleapis.com/${_GS_BUCKET}/$COMMIT_SHA/pipeline.zip
+        --pipeline_id ${_PIPELINE_ID} 
+        --version_name $COMMIT_SHA
+        --bucket_name ${_GS_BUCKET}
+        --gcr_address ${_GCR_PATH}"
+      ]
+    env:
+      - "CLOUDSDK_COMPUTE_ZONE=us-central1-a"
+      - "CLOUDSDK_CONTAINER_CLUSTER=v38"
+    id: "KaggleCreatePipelineVersionAndRun"
+
+images:
+  - "${_GCR_PATH}/kaggle_download:$COMMIT_SHA"
+  - "${_GCR_PATH}/kaggle_visualize_table:$COMMIT_SHA"
+  - "${_GCR_PATH}/kaggle_visualize_html:$COMMIT_SHA"
+  - "${_GCR_PATH}/kaggle_train:$COMMIT_SHA"
+  - "${_GCR_PATH}/kaggle_submit:$COMMIT_SHA"
+  
+  - "${_GCR_PATH}/kaggle_download:latest"
+  - "${_GCR_PATH}/kaggle_visualize_table:latest"
+  - "${_GCR_PATH}/kaggle_visualize_html:latest"
+  - "${_GCR_PATH}/kaggle_train:latest"
+  - "${_GCR_PATH}/kaggle_submit:latest"
+  
+
+substitutions:
+  _CODE_PATH: /workspace/kaggle
```

Comment:
I believe the _CODE_PATH here should be something like "/workspace/samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample" to match dockerfile locations. Please fix this one and other _CODE_PATH accordingly.

Reply:
Right. Will modify them.